truss-chains/truss_chains/framework.py

```diff
@@ -725,19 +728,10 @@ def _validate_dependencies(
         return dependencies
 
 
-def _validate_chainlet_cls(
-    cls: Type[definitions.ABCChainlet], location: _ErrorLocation
-) -> None:
-    if not hasattr(cls, definitions.REMOTE_CONFIG_NAME):
-        _collect_error(
```

Comment:
We can safely remove this since it'll never fire - the base abstract class [instantiates](https://github.com/basetenlabs/truss/blob/76468b37b4715cfb041e69fee68a4557a44e094e/truss-chains/truss_chains/definitions.py?plain=1#L505) this